.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -1,6 +1,7 @@
 <component name="InspectionProjectProfileManager">
   <profile version="1.0">
     <option name="myName" value="Project Default" />
+    <inspection_tool class="Eslint" enabled="true" level="WARNING" enabled_by_default="true" />
```

Comment:
We don't need to commit anything in the .idea folder as this is intellij metadata. I think it's best we remove these from git and add to gitignore reducing the size of PRs

Reply:
Yh that seems to be a mistake on my gitignore file on how i spelled the ignore idea one

Reply:
`.idea/` is good, but you need to remove the file from git, usually with `git rm` if I remember correctly (it won't physically delete the file)

Reply:
Ah my bad forgot to do that 

> usually with `git rm`

I think in cases like this you need to use the --cache like this `git rm --cached -r .idea` . Anyhow i think it should be fine now.